### PR TITLE
Feature/applics 813 search split term

### PR DESCRIPTION
**Pull request by johnmoscrip**
Changed the representations search to an AND when splitting search terms instead of OR and added a stop-word filter list to ignore common terms .

[https://pins-ds.atlassian.net/browse/APPLICS-813](https://pins-ds.atlassian.net/browse/APPLICS-813)

Updates to BO and NI repositories.

## Useful information to review or test

Example link - https://applications-service-dev.planninginspectorate.gov.uk/projects/TR010027/representations?searchTerm=Winthorpe+Primary+School&itemsPerPage=25.  Previously this would return a lot of unrelated results as it would be searching on "Winthorpe" OR "primary" OR "School".  Instead we target the inclusion of all of these terms.

Unit tests have been updated in line with the code updates.

NB.  The BO search still uses OR functionality for it's first name and last name fields due to the restrictive nature of having the data separated into two fields.  

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [X] I have performed a self-review of my own code
- [X] I have double checked this work does not include any hardcoded secrets or passwords
- [X] I have made corresponding changes to the documentation
- [X] I have provided details on how I have tested my code
- [X] I have referenced the ticket number above
- [X] I have provided a description of the ticket
- [X] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: packages/applications-service-api/__tests__/unit/repositories/representation.ni.repository.test.js [hunk 1]
```diff
@@ -65,12 +65,14 @@ describe('representation ni repository', () => {
 			// Act
 			await getRepresentationsWithCount(mockOptions);
 			// Assert
+
+			//Currently failing - needs investigation
```
[the remark this answers is not in the document]
Comment (johnmoscrip) [reply packages/applications-service-api/__tests__/unit/repositories/representation.ni.repository.test.js:69]: Ah thanks Ed.  I'd added and removed a few of those comments to tests that were failing but I fixed all of the tests.  Must have missed that comment.  I'll update it

File: packages/applications-service-api/src/utils/stopwords.js [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+		"you'll",
+		"you're",
+		"you've",
+		'your',
+		'yours',
+		'yourself',
+		'yourselves'
+	],
+
+	welsh: [
```
Comment (edpackard-work) [inline packages/applications-service-api/src/utils/stopwords.js:179]: will these be used? i.e. is there a future ticket to include Welsh stopwords?

**Review by edpackard-work — DISMISSED** [review]

File: packages/applications-service-api/src/utils/stopwords.js (hunk 2 above)
Comment (johnmoscrip) [reply packages/applications-service-api/src/utils/stopwords.js:179]: They were in use but i decided to just go with English in the end.  I was keeping those in there as an example of Welsh stop-words.  I was in two mind whether to remove or not?

Comment (edpackard-work) [reply packages/applications-service-api/src/utils/stopwords.js:179]: maybe leave them in with a comment on the ticket to possibly re-include in future for searches when language set to Welsh?

**Review by edpackard-work — APPROVED** [review]

**Pull request merged**